jans-cedarling/bindings/cedarling_python/example.py

```diff
@@ -0,0 +1,52 @@
+from cedarling_python import AuthzConfig, MemoryLogConfig, OffLogConfig, StdOutLogConfig
+from cedarling_python import PolicyStoreSource, PolicyStoreConfig, BootstrapConfig
+from cedarling_python import Cedarling
+
+
+authz_config = AuthzConfig(application_name="example_app_name")
+# we can also set value to as property
+# authz_config.application_name = "example_app_name2"
+
+# use log config to store logs in memory with a time-to-live of 120 seconds
+# by default it is 60 seconds
+log_config = MemoryLogConfig(log_ttl=100)
+# we can also set value to as property
+# log_config.log_ttl = 120
+
+# use disabled log config to ignore all logging
+# log_config = OffLogConfig()
+
+# use log config to print logs to stdout
+# log_config = StdOutLogConfig()
+
+# Create policy source configuration
+policy_raw_json = open("example_files/policy-store.json",
```

Comment:
Please close files after you're done reading them. Clean way to do this is:
```python
with open("example_files/policy-store.json", "r") as f:
    file_content = f.read()
# File is automatically closed after this and content is in variable file_content
```

Reply:
Thanks, done.